src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -165,8 +152,39 @@ public boolean equals(Object obj) {
                 && filteredGuests.equals(other.filteredGuests);
     }
 
+    //=========== Filtered Room List Accessors =============================================================
+
+    /**
+     * Returns an unmodifiable view of the list of {@code Room} backed by the internal list of
+     * {@code versionedAddressBook}
+     */
+    @Override
+    public ObservableList<Room> getFilteredRoomList() {
+        return FXCollections.unmodifiableObservableList(filteredRooms);
+    }
+
+    @Override
+    public void updateFilteredRoomList(Predicate<Room> predicate) {
+        requireNonNull(predicate);
+        filteredRooms.setPredicate(predicate);
+    }
+
+    /*
+    @Override
+    public RoomList getRoomList() {
+        return roomList;
+    }*/
     @Override
     public void checkoutRoom(RoomNumber roomNumber) {}
     @Override
     public void commitRoomList() {}
+
+    /*
+    @Override
+    public void displayRoomList(Predicate<Room> predicate) {
+        //Disable guest list
+        requireNonNull(predicate);
+        //filteredGuests.setPredicate(predicate);
+
+    }*/
```

Comment:
what are these two blocks of commented out code for

Reply:
Leftover. Removed!